website/static/js/profile.js

```diff
@@ -539,9 +581,21 @@ var SocialViewModel = function(urls, modes) {
             {label: 'Google Scholar', text: self.scholar(), value: self.scholar.url()}
         ];
     });
-
+        
+    self.profileWebsiteEmpty = ko.computed(function() {
```

Comment:
I use profileWebsiteEmpty to tell if the add another button should be in the view.  If there is a blank website, then you should not need to add another one.